docs/README.md

```diff
@@ -1,29 +1,22 @@
 # Unofficial Talon Docs
 
-Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete, but instead represent an 80/20 view of the API sufficient for getting started making extensions and using the system.
+Documenting my learning experiences with Talon, the speech-to-text system. These docs are not authorative or complete. After reading them you should be able to extend and use Talon.
```

Comment:
This repository makes no such guarantees. Also, the documents specifically were written in a style to avoid personally addressing the reader. In this case, when using "you should", instead of something like "one should".